var/spack/repos/builtin/packages/openspeedshop/package.py

```diff
@@ -83,8 +77,8 @@ class Openspeedshop(Package):
                          to point to target build.")
     variant('cuda', default=False,
             description="build with cuda packages included.")
-    variant('ptgf', default=False,
-            description="build with the PTGF based gui package enabled.")
+    variant('noqt3gui', default=False,
+            description="build without the build Qt3 gui package enabled.")
```

Comment:
Are there other options for the GUI? If not, I think this variant would be more clear if it was called `gui`.